metricbeat/metricbeat.template-es2x.json

```diff
@@ -625,14 +625,14 @@
                     },
                     "views": {
                       "properties": {
-                        "actual_disk_size": {
+                        "data_size": {
```

Comment:
The diff is confusing here. `disk_size` is acutally on like 635 and `data_size` was moved up here (alphabetical sorting).